### PR TITLE
remove duplicates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: channels/sa.m3u [hunk 1]
```diff
@@ -17,6 +15,8 @@ http://m.live.net.sa:1935/live/quran/playlist.m3u8
 https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/saudi_quran/hls1/saudi_quran.m3u8
 #EXTINF:-1 tvg-id="AlQuranAlKareemTV.sa" tvg-name="Al Quran Al Kareem TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/0aMNwQa.png" group-title="Religious",Al Quran Al Kareem TV (Mecca) (1080p)
 https://iptv--iptv.repl.co/Arabic/Quran_TV/
+#EXTINF:-1 tvg-id="AlQuranAlKareemTV.sa" tvg-name="Al Quran Al Kareem TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/0aMNwQa.png" group-title="Religious",Al Quran Al Kareem TV (Mecca)
+https://5b18be6964c2f.streamlock.net:1935/live/_definst_/quran/playlist.m3u8
```
Comment (Dum4G) [inline channels/sa.m3u:19]: Returns playlist but can't confirm it working

File: channels/sa.m3u [hunk 2]
```diff
@@ -79,24 +71,12 @@ https://hiplayer.hibridcdn.net/t/rotana-lbc.m3u8
...
 #EXTINF:-1 tvg-id="PanoramaFM.sa" tvg-name="Panorama FM" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Music",Panorama FM (1080p)
 https://shls-panoramafm-prod-dub.shahid.net/out/v1/66262e420d824475aaae794dc2d69f14/index.m3u8
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV
-http://5b18be6964c2f.streamlock.net:1935/live/_definst_/quran/chunklist_w2066488187.m3u8
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV
-http://m.live.net.sa:1935/live/quran/chunklist_w232396531.m3u8?v=2957056943
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV
-http://m.live.net.sa:1935/live/quran/chunklist_w994500087.m3u8
-#EXTINF:-1 tvg-id="QuranTV.sa" tvg-name="Quran TV" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",Quran TV (720p)
-http://m.live.net.sa:1935/live/quran/gmswf.m3u8
```
Comment (Dum4G) [inline channels/sa.m3u:99]: works

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/sa.m3u (hunk 2 above)
Comment (tilfizyawn) [reply channels/sa.m3u:99]: It's a duplicate of http://m.live.net.sa:1935/live/quran/playlist.m3u8

File: channels/sa.m3u (hunk 1 above)
Comment (tilfizyawn) [reply channels/sa.m3u:19]: Working https://streamtest.in/logs/https-5ab29cc78f681-streamlock-net-makkahtv-livestream-playlist-m3u8

**Review by Dum4G — APPROVED** [review]

**Pull request merged**